### DISCUSSION TITLE
use multiple apps as a single app using its multi-zones feature.

### DISCUSSION THREAD

**Discussion by tdpksingh** [vercel/vercel#9195] [Help]
i am trying to use multiple apps as a single app using its multi-zones feature. its working perfectly in my local machine, the redirection is not working in vercel after deployment, its giving 404 error, my configs looks like this ->

{ async rewrites() {
    return [
        {
          source: '/:path*',
          destination: `/:path*`,
        },
        {
          source: '/sell',
          destination: `${NEXT_PUBLIC_APP_URL_SELL}/sell`,
        },
        {
          source: '/sell/:path*',
          destination: `${NEXT_PUBLIC_APP_URL_SELL}/sell/:path*`,
        },

      ]
  },
  reactStrictMode: true,
  poweredByHeader: false,
  generateEtags: false,
  ...nextTranslate(),
  basePath: '/buy'
}

**Comment by snghadv21** [comment]
I am following you up, as I am facing the same issue.
